package.json

```diff
@@ -50,6 +50,7 @@
     "i18next": "^12.1.0",
     "lambda-log": "^2.0.0",
     "lodash": "^4.13.1",
+    "querystring": "^0.2.0",
```

Comment:
this module is available inside core node https://devdocs.io/node~8_lts/querystring#querystring_querystring_stringify_obj_sep_eq_options